templates/search/blocks/search_filters.html

```diff
@@ -54,7 +54,9 @@ <h4 class="search-filters__accordion-section-heading">Date</h4>
 
             {% include "search/includes/filter.html" with field=form.topic %}
 
-            {% include "search/includes/filter.html" with field=form.level %}
+            {% if form.group.value != 'nonTna' %}
```

Comment:
Making decisions on hardcode data is tricky, so what happened if the data changes from nonTna to nontna , this code will no longer work,

Reply:
Yes, its tricky. 

This is key data, ` "key": "nonTna",` in response and ideally that must not change. Its like a specification.
https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/ciim/constants.py#L60

if `nontna`, then the bucket for `Records for Other Archives` would not work as expected.

As such, all areas in search where this key is used would fail.

There are many such keys.- buckets - `tna, digitised,etc` 

I'm happy to do a lower comparison if otherwise.

Reply:
The issue here is that will fail silently, and since it is a constant I suggest making it a constant in the relevant search view class, at least the change will be in one place. Would you be able to write a negative test, to handle invalid value, and then log the error?

Reply:
> The issue here is that will fail silently
Isn't this relevant then `We did not find any results for your search' ?

> Would you be able to write a negative test, to handle invalid value, and then log the error?
Can it be more specific? Is this related to the comparison in the template? 
I am not sure of error logging for templates? Can you give me any pointers?

Reply:
Yes, sorry , so if you move the 'nonTna' to the class and make it a constant ,  then you would be able to validate the value against what may come from the API or any source before returning the response, this way will know why it had failed by looking into log. in general hard-coded values must be constant ,also the advantage of making a constant then you will be able to change that constant value in one place . let's say the specification has changed for any reason then here you only change the constant once to the new value.

Reply:
Can we please have a discussion?